_internal/enums.ts

```diff
@@ -0,0 +1,31 @@
+export const enum InternalUtils {
```

Comment:
maybe explain why we need to have this as comment here, to help contributors understand better this is for size compression purpose